File: Day One/java phasegate One/TaskSeven.java
import java.util.Scanner;
	public class TaskSeven {
	
		public static void main(String[] args) { 
		Scanner scanner = new Scanner(System.in);
	
		int sum = 0;
		for(int counter = 1; counter <=10; counter ++) { 
			System.out.print("Enter Scores " + counter + ": ");
			int scores = scanner.nextInt();
			if(scores % 2 == 0){ sum += scores;}
		}
		int average = sum / 10;
		System.out.println("The average of the even scores collected is: " + average);
		System.out.println("The sum of the even scores collected is: " + sum);
		}
}
 

File: Day One/python phasegate One/Task_seven.py
sum = 0;
for number in range(1, 11):
	scores = int(input("Enter scores: ")) 
	if scores % 2 == 0: 
		sum += scores
average = sum / 10
print(f"The average of the scores is: {average}")
print(f"The sum of the scores is: {sum}")


	
 